### PR TITLE
twitter roberta base sentiment example

### PR DESCRIPTION
**Pull request by plutopulp**
## Checklist:
- [ ] Version bumped

Added:
- Code example for uploading and running the pipeline: twitter-roBERTa-base-sentiment

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pipeline-ai"
-version = "1.0.6"
+version = "1.0.7"
```
Comment (paulcjh) [inline pyproject.toml:3]:
```suggestion
```

**Review by paulcjh — DISMISSED** [review]
Just gotta remove the version increment, then LGTM, nice!

**Comment by plutopulp** [comment]
> Just gotta remove the version increment, then LGTM, nice!

Oh yeah makes sense, no point in that here. Thanks

**Review by paulcjh — APPROVED** [review]
Awesome :)

**Pull request merged**